packages/tasit-sdk/src/Decentraland.test.js

```diff
@@ -110,12 +110,19 @@ describe("Decentraland", () => {
         );
       });
 
-      beforeEach("", async () => {
+      beforeEach("check Gnosis Safe balance", async () => {
         gnosisSafe = new GnosisSafe(GNOSIS_SAFE_ADDRESS);
 
-        // Expect an already-funded Gnosis Safe wallet
-        await confirmEtherBalances(provider, [GNOSIS_SAFE_ADDRESS], [ONE]);
-        await confirmBalances(mana, [GNOSIS_SAFE_ADDRESS], [ONE_HUNDRED]);
+        // Expect an already funded Gnosis Safe wallet
+        let etherBalance = await provider.getBalance(GNOSIS_SAFE_ADDRESS);
+        etherBalance = bigNumberify(etherBalance);
+        const one = bigNumberify(ONE);
+        expect(etherBalance.gte(one)).to.be.true;
+
+        let manaBalance = await mana.balanceOf(GNOSIS_SAFE_ADDRESS);
```

Comment:
Why did you move away from the confirmBalances strategy here? Because of the `greater than or equal to`? We could have a separate convenience function for that or an extra param for the existing function if you want.

Reply:
Yes, sure!

Reply:
Done.

Reply:
Thanks! Resolving